config/routes.rb

```diff
@@ -21,6 +21,7 @@
     end
     resources :placements_preview, only: [:show]
     get "rollups_demo", to: "podcasts#rollups_demo"
+    resource :metrics, controller: :podcast_metrics
```

Comment:
I suspect eventually this will be several routes?  But this works just to get things stubbed in.